fluent-package/Rakefile

```
@@ -537,31 +538,31 @@ class BuildTask
       desc "Install additional .bat files for Windows"
       task :win_batch_files do
         ensure_directory(staging_bindir)
-        cp("msi/assets/#{PACKAGE_NAME}-prompt.bat", td_agent_staging_dir)
-        cp("msi/assets/#{PACKAGE_NAME}-post-install.bat", staging_bindir)
-        cp("msi/assets/#{PACKAGE_NAME}.bat", staging_bindir)
-        cp("msi/assets/#{PACKAGE_NAME}-gem.bat", staging_bindir)
-        cp("msi/assets/#{PACKAGE_NAME}-version.rb", staging_bindir)
+        cp("msi/assets/#{SERVICE_NAME}-prompt.bat", td_agent_staging_dir)
+        cp("msi/assets/#{SERVICE_NAME}-post-install.bat", staging_bindir)
+        cp("msi/assets/#{SERVICE_NAME}.bat", staging_bindir)
+        cp("msi/assets/#{SERVICE_NAME}-gem.bat", staging_bindir)
+        cp("msi/assets/#{SERVICE_NAME}-version.rb", staging_bindir)
       end
 
       desc "Create systemd unit file for Red Hat like systems"
       task :rpm_systemd do
-        dest =  File.join(STAGING_DIR, 'usr', 'lib', 'systemd', 'system', PACKAGE_NAME + ".service")
+        dest =  File.join(STAGING_DIR, 'usr', 'lib', 'systemd', 'system', PACKAGE_DIR + ".service")
         params = {pkg_type: "rpm"}
         render_systemd_unit_file(dest, template_params(params))
         render_systemd_environment_file(template_params(params))
       end
 
       desc "Create sysv init file for Red Hat like systems"
       task :rpm_sysvinit do
-        dest =  File.join(STAGING_DIR, 'etc', 'init.d', PACKAGE_NAME)
+        dest =  File.join(STAGING_DIR, 'etc', 'init.d', PACKAGE_DIR)
```

Should it be `SERVICE_NAME`?

BTW we no longer support SySV init based OS, so it should be removed (in another PR?).

---

It's out of scope in this PR (It should be removed in  another PR)

---

O.K. Cloud you just replace with `SERVICE_NAME`?
```suggestion
        dest =  File.join(STAGING_DIR, 'etc', 'init.d', SERVICE_NAME)
```